src/objects/DataImportBatch__c.object

```diff
@@ -224,6 +234,16 @@
         <type>Text</type>
         <unique>false</unique>
     </fields>
+    <fields>
```

Comment:
I believe we removed this?

Reply:
Yep! Not sure why it wasn't carried over.

Reply:
I think I deleted the field from template but maybe you added it for batch?